Ginger/GingerCoreNET/Reports/HTMLReportConfiguration.cs

```diff
@@ -71,6 +72,10 @@ public bool IsDefault
         [IsSerializedForLocalRepository]
         public bool ShowAllIterationsElements { get { return mShowAllIterationsElements; } set { if (mShowAllIterationsElements != value) { mShowAllIterationsElements = value; OnPropertyChanged(nameof(ShowAllIterationsElements)); } } }
 
+        string mExecutionCalculationLevel;
+        [IsSerializedForLocalRepository]
+        public string ExecutionCalculationLevel { get { return mExecutionCalculationLevel; } set { if (mExecutionCalculationLevel != value) { mExecutionCalculationLevel = value; OnPropertyChanged(nameof(ExecutionCalculationLevel)); } } }
```

Comment:
1- call the field "ExecutionStatisticsCountBy"
2- it value type should be enum "eExecutionStatisticsCountBy" as i detailed in previous comment